spotinst/commons/managed_instance_aws_compute_launchspecification/fields_spotinst_managed_instances_aws_compute_launchspecification.go

```diff
@@ -755,21 +752,12 @@ func isBase64Encoded(data string) bool {
 	return err == nil
 }
 
-func hashKV(v interface{}) int {
```

Comment:
make sure it pass backward compatibility tests

Reply:
done and verified